src/app/app.css

```
@@ -39,7 +39,7 @@ limitations under the License.
 
 .cryostat-app .cryostat-text {
   margin-top: -0.5em !important;
-  font-family: "Montserrat", sans-serif;
+  font-family: 'Montserrat', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Montserrat" (font-family-name-quotes)

(font-family-name-quotes)